metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -109,8 +111,15 @@ def version() -> None:
 
 
 @datahub.command()
+@click.option(
+    "--use-password",
+    type=bool,
+    is_flag=True,
+    default=False,
+    help="If passed then uses password to initialise token.",
+)
 @telemetry.with_telemetry()
-def init() -> None:
+def init(use_password: bool = False) -> None:
```

Comment:
can we update these docs https://datahubproject.io/docs/next/cli/#init

Reply:
done